examples/config-api/config_api.description

```diff
@@ -1 +1,21 @@
-//The `ballerina/config` module provides an API to read configurations from various sources such as files, environment variables and CLI parameters.
+// The Ballerina Config API allows you to look up values from configuration
+// files, CLI parameters, and environment variables. The precedence order for
+// configuration resolution is as follows: <br/>
+//
+// * CLI parameters
+// * Environment variables
+// * Configuration files
+//
+// If a specific configuration defined in the file is also defined as an
+// environment variable, the environment variable takes precedence. Similarly,
+// if the same is set as a CLI parameter, it replaces the environment
+// variable value.
+//
+// The configuration file is in the TOML(v0.4) format.
```

Comment:
Remove this line. At least for now. Need to verify which version we support in 1.2.x

Reply:
Removed.